tao_compiler/mlir/disc/BUILD

```diff
@@ -842,6 +842,7 @@ cc_library(
         ":qconv_custom_call",
         ":sparse_custom_call",
         ":topk_custom_call",
+        "//tensorflow/compiler/xla/mlir_hlo:lhlo",
```

Comment:
why it's needed? won't dependencies above like `lmhlo_disc` bring it into depencency tree?

Reply:
in order to insert `lmhlo::CopyOp`.